pallets/migrations/src/lib.rs

```diff
@@ -135,9 +143,22 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
 		fn build(&self) {
+			//TODO Does it make sense to configure migrations at genesis? Would they even be run?
```

Comment:
We do want to be able to tell `pallet-migrations` that our genesis effectively includes migrations already, but maybe this should be implicit (a brand new chain implicitly has all configured migrations completed).

Reply:
Oh, I get it now. And I agree. So this is for the case where we launch Moonbeam with a genesis runtime that already "includes" some migrations that were performed on moonriver for example. And here "includes" doesn't mean that they were actually executed on chain, but more that they shouldn't be executed in the future because that would corrupt state.